pig/src/test/java/org/jboss/pnc/bacon/pig/impl/addons/scanservice/ScanServiceDTOTest.java

```diff
@@ -0,0 +1,169 @@
+
+public class ScanServiceDTOTest {
+
+    // private PncBuild pncBuild;
+    private Map<String, PncBuild> builds;
+    static EasyRandom easyRandom = new EasyRandom();
+
+    @BeforeEach
+    public void init() {
+        builds = new HashMap<>();
+
+        for (int i = 0; i < 5; i++) {
+            PncBuild b = easyRandom.nextObject(PncBuild.class);
+            b.setInternalScmUrl("https://github.com/michalszynkiewicz/demo-bar");
+            b.setScmRevision("1.0.1");
+            builds.put(easyRandom.nextObject(String.class), b);
+        }
+
+    }
+
+    public String getSerializedDTO(ScanServiceDTO ss) throws JsonProcessingException {
+        return new ObjectMapper().writeValueAsString(ss);
+    }
+
+    public static ScanServiceDTO dummyScanService() {
+
+        ScanServiceDTO ss = new ScanServiceDTO();
+        ss.setProductId("23");
+        ss.setEventId(ScanServiceDTO.EVENT_ID_DFT_VALUE);
+        ss.setIsManagedService(ScanServiceDTO.IS_MANAGED_SERVICE_DFT_VALUE);
+        ss.setCpaasVersion(ScanServiceDTO.CPAAS_VERSION_DFT_VALUE);
+        ss.setJobUrl(ScanServiceDTO.JOB_URL_DFT_VALUE);
+        return ss;
+    }
+
+    @Test
+    public void fromPncBuild() {
+        ScanServiceDTO ss = new ScanServiceDTO(
+                "productId",
+                "eventId",
+                true,
+                "cpaasVersion",
+                "jobUrl",
+                null,
+                null,
+                builds);
+        try {
+            String jsonDTO = getSerializedDTO(ss);
+            System.out.println("Test fromPncBuild: " + jsonDTO);
+        } catch (JsonProcessingException e) {
+            fail("ScanSerivce can't be instantiated as json");
+        }
+    }
+
+    @Test
+    public void fromPncBrewGit() {
+        List<Integer> brewBuilds = Arrays.asList(easyRandom.nextInt(10000), easyRandom.nextInt(10000));
+        Map<String, String> scmUrls = Stream
+                .of(
+                        new String[][] {
+                                { "scmUrl", "https://github.com/michalszynkiewicz/demo-bar" },
+                                { "scmRevision", "1.0.1" }, })
+                .collect(Collectors.toMap(data -> (String) data[0], data -> (String) data[1]));
+
+        ArrayList scmUrlsL = new ArrayList<Map<String, String>>();
+        scmUrlsL.add(scmUrls);
+
+        ScanServiceDTO ss = new ScanServiceDTO(
+                "productId",
+                "eventId",
+                true,
+                "cpaasVersion",
+                "jobUrl",
+                brewBuilds,
+                scmUrlsL,
+                builds);
+        try {
+            String jsonDTO = getSerializedDTO(ss);
+            System.out.println("Test fromPncBuild: " + jsonDTO);
+        } catch (JsonProcessingException e) {
+            fail("ScanSerivce can't be instantiated as json");
+        }
+    }
+
+    @Test
+    public void brewID() {
+        ScanServiceDTO ss = this.dummyScanService();
```

Comment:
Either dummyScanService should be accessed in a non-static way or it should setup an object on a per-test basis.